build/ProjectReunion-BuildFoundation.yml

```diff
@@ -29,10 +29,6 @@ jobs:
         buildConfiguration: 'Release'
         normalizedConfiguration: 'fre'
         PGOBuildMode: 'Optimize'
-      Release_Arm:
-        buildPlatform: 'arm'
-        buildConfiguration: 'Release'
-        normalizedConfiguration: 'fre'
       Release_Arm64:
```

Comment:
Why do Release_x86 + Release_x64 have PGOBuildMode:Optimize and Release_Arm64 doesn't?

Reply:
Sounds like optimize is good to have for a release build. Should I add it to Arm64?

Reply:
Good question to save as an issue / bug. I wouldn't get hung up on it in this review.